### PR TITLE
v6.9.2

### PR DESCRIPTION
**Pull request by MBilalShafi**
### Release the packages

- [ ] Checkout the last version of the working branch
- [ ] Make sure you have the latest dependencies installed: `yarn`.
- [ ] Build the packages: `yarn release:build`.
- [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
- [ ] Push the newly created tag: `yarn release:tag`.

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

- [ ] Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```bash
git push upstream master:docs-v6 -f
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-v6)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-v6` deployment.

### Publish GitHub release

- [ ] After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,60 @@
...
+_Jul 6, 2023_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Auto-scroll when making range selection (#8661) @m4theushw
+- 🥧 Add pie chart component
+
+  They are fresh from the code editor. You can visit [pie charts docs](https://mui.com/x/react-charts/pie/) or their [demo page](https://mui.com/x/react-charts/pie-demo/).
+
+![image](https://github.com/mui/mui-x/assets/45398769/20f31f44-9fb0-49c1-b77c-71bdc5afea0b)
```
Comment (cherniavskii) [inline CHANGELOG.md:17]: Slightly better image quality :)
```suggestion
![image](https://github.com/mui/mui-x/assets/13808724/fe908c45-803c-4316-b913-dbd2f9f0551e)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,60 @@
...
+
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### `@mui/x-data-grid@v6.9.2` / `@mui/x-data-grid-pro@v6.9.2` / `@mui/x-data-grid-premium@v6.9.2`
+
+#### Changes
+
+- [DataGrid] Fix `RangeError` when flex columns have duplicated `field` property (#9554) @cherniavskii
```
Comment (cherniavskii) [inline CHANGELOG.md:26]: I just realized that the PR title needed an update 🙂
```suggestion
- [DataGrid] Fix `RangeError` when using flex columns (#9554) @cherniavskii
```

**Review by cherniavskii — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,60 @@
...
+
+#### Changes
+
+- [charts] Add pie chart component (#9395) @alexfauquette
+
+### Docs
+
+- [docs] Add pickers playground (#9164) @LukasTy
+- [docs] Fix API links for pickers (#9573) @alexfauquette
+- [docs] Fix typos in pagination documentation page (#9332) @RatherBeLunar
```
Comment (alexfauquette) [inline CHANGELOG.md:49]:
```suggestion
- [docs] Fix demos with `ToggleButtonGroup` (#9548) @flaviendelangle
- [docs] Fix typos in pagination documentation page (#9332) @RatherBeLunar
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,60 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.9.2
+
+_Jul 6, 2023_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Auto-scroll when making range selection (#8661) @m4theushw
+- 🥧 Add pie chart component
```
Comment (alexfauquette) [inline CHANGELOG.md:13]:
```suggestion

- 📚 New page: Components lifecycle (#8372) @flaviendelangle
 
  Clarify pickers events and value updates in a [single docs page](https://mui.com/x/react-date-pickers/lifecycle/).
  
- 🥧 Add pie chart component
```

**Review by alexfauquette — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,67 @@
...
+
+- 📚 New page: Components lifecycle (#8372) @flaviendelangle
+
+  Clarify pickers events and value updates in a [single docs page](https://mui.com/x/react-date-pickers/lifecycle/).
+
+- 🥧 Add pie chart component
+
+  They are fresh from the code editor. You can visit [pie charts docs](https://mui.com/x/react-charts/pie/) or their [demo page](https://mui.com/x/react-charts/pie-demo/).
+
+![image](https://github.com/mui/mui-x/assets/13808724/fe908c45-803c-4316-b913-dbd2f9f0551e)
```
Comment (cherniavskii) [inline CHANGELOG.md:22]:
```suggestion
<img width="380" alt="pie-charts" src="https://github.com/mui/mui-x/assets/13808724/fe908c45-803c-4316-b913-dbd2f9f0551e">
```
To keep avoid expanding beyond original size

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,67 @@
...
+
+- 📚 New page: Components lifecycle (#8372) @flaviendelangle
+
+  Clarify pickers events and value updates in a [single docs page](https://mui.com/x/react-date-pickers/lifecycle/).
+
+- 🥧 Add pie chart component
+
+  They are fresh from the code editor. You can visit [pie charts docs](https://mui.com/x/react-charts/pie/) or their [demo page](https://mui.com/x/react-charts/pie-demo/).
+
+<img width="380" alt="pie-charts" src="https://github.com/mui/mui-x/assets/13808724/fe908c45-803c-4316-b913-dbd2f9f0551e">
```
Comment (MBilalShafi) [inline CHANGELOG.md:22]:
```suggestion
  <img width="380" alt="pie-charts" src="https://github.com/mui/mui-x/assets/13808724/fe908c45-803c-4316-b913-dbd2f9f0551e">
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,67 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.9.2
+
+_Jul 6, 2023_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
```
Comment (MBilalShafi) [inline CHANGELOG.md:10]:
```suggestion
We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
```

**Pull request merged**